portfolio/src/main/webapp/index.html

```diff
@@ -113,14 +113,22 @@ <h3>Gallery</h3>
       <div id="Comments" class="tab-info">
         <h3>Comments</h3>
         <p>Submit your own comment below!</p>
-        <form action="/data" method="POST">
-          <label for="author">Name:</label><br> 
-          <input type="text" id="author" name="author"><br></br>
-          <label for="message">Message:</label><br>
-          <input type="text" id="message" name="message"></br></br>
-          <input type="submit" id="submit"/>
-        </form>
-        <input type="number" id="max-comments" name="max-comments" min="0" value="5" onchange="fetchComments(value)"></br></br>
+        <div id="input-container">
+          <div id="comment-input">
+            <form action="/data" method="POST">
+              <label for="author-input">Name:</label></br>
```

Comment:
Renaming such variable is not required for the purpose of this particular PR. If you think this is required, then please do so in a separate PR, explaining the rationale behind renaming.